### PR TITLE
[Improve] record more info of application log

### LINKED ISSUE #2473: [Feature] improve application log
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

application log in streampark records these state:
 
 - the success and fail state of start operation
 - the fail state of start/cancel operation
 
But it missed the release operation state. And the application log does not have name field, so we cannot tell what the operation is in web-ui. I propose that we should record more info of user operation:

- record the result of Release operation;
- record the successful result of Savepoint/Cancel operation;
- add `Operation Name` column in Option Log tab;
- add `delete` button in Option Log tab;
- add name field for ApplicationLog;

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhoulii**
## What changes were proposed in this pull request

Issue Number: close #2473 

## Brief change log

before:
before

after:
after

## Verifying this change

- *Manually verified the change by testing locally.* 

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / **no**)

### CODE REVIEW

File: streampark-console/streampark-console-webapp/src/views/flink/app/components/AppDetail/DetailTab.vue [hunk 1]
```diff
@@ -430,6 +465,12 @@
       <TabPane key="6" tab="Option Logs" v-if="tabConf.showOptionLog">
         <BasicTable @register="registerLogsTable">
           <template #bodyCell="{ column, record }">
+            <template v-if="column.dataIndex == 'optionName'">
+              <Tag color="#52c41a" v-if="record.optionName == 0"> Release </Tag>
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/app/components/AppDetail/DetailTab.vue]: We recommend using enumerations to standardize the meaning of data. Here, you can refer to the 220 lines of code in streampark-console-webapp/src/views/flink/app/View. vue. Of course, writing this currently does not affect program operation

Comment (zhoulii) [reply streampark-console/streampark-console-webapp/src/views/flink/app/components/AppDetail/DetailTab.vue]: nice suggestion, done.

Comment (zhoulii) [reply streampark-console/streampark-console-webapp/src/views/flink/app/components/AppDetail/DetailTab.vue]: nice suggestion, done.

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**